packages/run-protocol/src/core-proposal.js

```diff
@@ -195,20 +179,25 @@ const roleToGovernanceActions = harden({
   client: {},
 });
 
-export const getRunProtocolManifest = (
+export const getManifestForRunProtocol = (
   { restoreRef },
-  { ROLE = 'chain', installKeys },
+  { ROLE = 'chain', installKeys, vaultFactoryControllerAddress },
 ) => {
   return {
     manifest: roleToGovernanceActions[ROLE],
     installations: {
       runStake: restoreRef(installKeys.runStake),
       amm: restoreRef(installKeys.amm),
-      vaultFactory: restoreRef(installKeys.vaultFactory),
+      VaultFactory: restoreRef(installKeys.vaultFactory),
```

Comment:
capitalization change necessary?

Reply:
this change is necessary for consistency with other places.

Perhaps they should all be changed to `vaultFactory`, but that involves the dApps too, so... not today.